code-gen-projects/input/good/nested_struct/valid_optional_fields.ion

```diff
@@ -0,0 +1,9 @@
+// nested struct with some optional fields that are not provided
+{
+A: "hello",
+// B: 12, // since `B` is optional field, this is a valid struct
+C: {
+D: false,
+E: [1, 2, 3]
+}
+}
```

Comment:
Could you add 2- or 4-space indentation to these `.ion` files to make the nesting clearer?

Reply:
It looks like these files are still unformatted.